source/people/aaswenson.rst

```diff
@@ -0,0 +1,29 @@
+Alexander Swenson
+=============
+
+Contact Information
+-------------------
+| `E-mail <aaswenson@wisc.edu>`_
+
+----
```

Comment:
I meant remove the dashed lines (----). An blank line for division of blocks is necessary, the dashed line is not. Sorry for the confusion.